code/modules/mob/living/carbon/carbon.dm

```diff
@@ -543,6 +547,10 @@
 			see_invisible = G.invis_override
 		else
 			see_invisible = min(G.invis_view, see_invisible)
+		if(!isnull(G.lighting_alpha) && hud_used)
+			var/obj/screen/plane_master/lighting/L = hud_used.plane_masters["[LIGHTING_PLANE]"]
+			if (L)
+				L.alpha = G.lighting_alpha

```

Comment:
I could make it merge the two, I added this before adding eyes, and forgot to take into account when both exist.

I'll also make it default properly.